service/history/timerQueueActiveProcessor.go

```diff
@@ -430,10 +430,17 @@ ExpireActivityTimers:
 			}
 		}
 
+		domainEntry, err := t.shard.GetDomainCache().GetDomainByID(msBuilder.GetExecutionInfo().DomainID)
+		if err != nil {
+			return &workflow.InternalServiceError{Message: "Unable to get domain from cache by domainID."}
```

Comment:
nit: first letter lower case for the error message.

Reply:
OK